monkey/monkey_island/cc/ui/src/components/ui-components/MissingBinariesModal.js

```diff
@@ -0,0 +1,63 @@
+import {Modal} from 'react-bootstrap';
+import React from 'react';
+import {GridLoader} from 'react-spinners';
+
+
+class MissingBinariesModal extends React.PureComponent {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showModal: this.props.showModal,
+      errorDetails: this.props.errorDetails
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props !== prevProps) {
+      this.setState({
+        showModal: this.props.showModal,
+        errorDetails: this.props.errorDetails
+      })
+    }
+  }
+
+  render = () => {
+    return (
+      <Modal show={this.state.showModal} onHide={() => this.props.onClose()}>
+        <Modal.Body>
+          <h3>
+            <div className='text-center'>Uh oh...</div>
+          </h3>
+          <div style={{'marginTop': '1em', 'marginBottom': '1em'}}>
+            <p className="alert alert-warning">
+              <i className="glyphicon glyphicon-info-sign" style={{'marginRight': '5px'}}/>
+              Some Monkey binaries are not found where they should be...<br/>
+              Try downloading them from <a href="https://github.com/guardicore/monkey/releases/latest" target="blank">here</a>,
+              at the bottommost section titled "Assets".
```

Comment:
Ideally we should add "... and place them in `monkey/monkey_island/cc/binaries` directory."

Reply:
I wish I could make the directory path look like `that`, I used italics instead 🙄

Reply:
You can do `this` using `import marked from 'marked';` and then styling your text using  Markdown. You can see an example of this in `monkey_island/cc/ui/src/components/attack/techniques/MitigationsComponent.js`.

https://github.com/markedjs/marked

Reply:
Alternatively, use a `div` with the `className="label"`

Reply:
I simply used `<code>` :)
![image](https://user-images.githubusercontent.com/48174048/83195787-75d51100-a143-11ea-999c-ef818cc1f964.png)